### PR TITLE
Build debian installer using buildkite

### PR DESCRIPTION
**Pull request by mrpau-eduard**
## Summary
Build the debian installer using docker in our buildkite.

## TODO

- [x] Debian installer
- [x] ka-lite-raspberry-pi_*-0ubuntu1_all.deb
- [x] ka-lite_*-0ubuntu1_all.deb
- [x] ka-lite-bundle_*-0ubuntu1_all.deb
- [ ] Upload to Google Cloud

## Issues addressed
#5473

### CODE REVIEW

**Comment by mrpau-eduard** [comment]
Don't merge yet @benjaoming I will create a separate script for the windows installer build. 
Thanks :)

File: .buildkite/build_debian.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    echo ".. Abort!  Error/s encountered running 'mk-build-deps'."
+    exit 1
+fi
+
+((STEP++))
+echo "$STEP/$STEPS. Now Running  '$DEBUILD_CMD'..."
+
+cd "$KA_LITE_DIR"
+$DEBUILD_CMD
+if [ $? -ne 0 ]; then
```
Comment (benjaoming) [inline .buildkite/build_debian.sh]: is it possible instead of these checks to have `set -e`?

**Comment by benjaoming** [comment]
@mrpau-eduard to avoid having content packs in the github repo, we run `debian/prepare_assessment_items.sh` which downloads it in case it isn't there.

But is there a way to cache the assessment items in Buildkite? Maybe some mechanism that all of the builds can share?

**Comment by benjaoming** [comment]
Closing, but will refer to this PR later, when creating configuration for Kolibri, and in case we prioritize it for KA Lite.

**Pull request closed without merging**